data/human/hails.txt

```diff
@@ -1803,6 +1876,60 @@ phrase "friendly navy"
 		"in the Republic"
 	word
 		"!"
+		
+phrase "friendly navy"
+	word
+		"Dying"
+		"Getting wounded"
+		"Taking a hit"
+	word
+		" "
+	word
+		"during"
+		"in"
+	word
+		" "
+	word
+		"a firefight"
+		"battle"
+		"space battle"
+		"war"
+	word
+		" is "
+	word
+		"heroic"
+		"honourable"
+		"respectable"
+		"valuable"
+		"well rewarded"
```

Comment:
These last two seem out of place to me.

"Taking a hit during a war is valuable"
"Dying in war is well-regarded"

Uh no it's not lol

Reply:
The last one was `well rewarded`, not `well regarded`. I removed them.